src/osg_display/gratia_datasource.py

```diff
@@ -260,84 +287,26 @@ def query_transfers(self):
         month_results = [i[0] for i in all_results]
         count_results = [i[1] for i in all_results]
         hour_results = [i[2] for i in all_results]
-        num_results = int(self.cp.get("Gratia", "months"))
+        num_results = int(self.cp.get("Gracc", "months"))
         month_results = month_results[-num_results:]
         count_results = count_results[-num_results:]
         hour_results = hour_results[-num_results:]
 
-	#write the data to cache file
-	pickle_f_handle = open(self.cache_data_file_name, "w")
-	cPickle.dump(all_results, pickle_f_handle)
-	pickle_f_handle.close()
+        #write the data to cache file
+        pickle_f_handle = open(self.cache_data_file_name, "w")
+        cPickle.dump(all_results, pickle_f_handle)
+        pickle_f_handle.close()
```

Comment:
Yeah......  I had to ask myself that too when I was reviewing the changes.  But i decided it was a worthy change.

Unfortunately, a bunch of the functions throughout the file go back and forth between spaces and tabs within individual functions -- which is really not proper in python.  I didn't try to fix it everywhere, either; i just did it in the functions that i was updating in this PR.  But i did make a separate commit for the whitespace-only changes, because i am really not a fan of putting functional changes and whitespace cleanup into the same commit.